lms/views/admin/organization.py

```diff
@@ -105,17 +105,14 @@ def toggle_organization_enabled(self):
         renderer="lms:templates/admin/organizations.html.jinja2",
     )
     def search(self):
-        if public_id := self.request.params.get("public_id"):
-            try:
-                orgs = [self.organization_service.get_by_public_id(public_id)]
-            except InvalidPublicId as err:
-                self.request.session.flash(str(err), "errors")
-                orgs = []
-
-        else:
+        try:
             orgs = self.organization_service.search(
-                name=self.request.params.get("name", "").strip()
+                name=self.request.params.get("name"),
```

Comment:
We don't need the strip any more as the new full text search processing stuff takes care of it